### ISSUE TITLE
MPJPE Loss in training

### ISSUE THREAD

**Issue by jder1234** [Arthur151/ROMP#256]
Hi, @Arthur151 , thanks for your excellent work.

I carried out my ROMP training with the latest code on all the datasets listed in the README and got the training log: 
[V1_hrnet_h36m,mpiinf,coco,mpii,lsp,muco,crowdpose_g0,1,2,3.log](https://github.com/Arthur151/ROMP/files/8784117/V1_hrnet_h36m.mpiinf.coco.mpii.lsp.muco.crowdpose_g0.1.2.3.log)

But a massive gap was found between my "MPJPE" loss with what you posted in issue #121.  My "MPJPE" loss was around 30 while yours was around 110.  

Did I miss something?

**Comment by Arthur151** [comment]
Hi, thanks for your kind words.
The log I posted was for someone who was interested in training from scratch. 
Normally, we just need to train from pre-trained backbone to save time and enjoy fatster converge. 
Best.

**Comment by Arthur151** [comment]
Yes, the training code would be released within a month.
